src/Network/Wai/Log/Options.hs

```diff
@@ -93,17 +93,15 @@ defaultLogRequest uuid req =
 --
 -- Time is in seconds as that is how 'NominalDiffTime' is treated by default
 defaultLogResponse :: UUID -> Request -> Response -> Value -> ResponseTime -> [Pair]
-defaultLogResponse uuid req resp responseBody time =
-    [ "request_uuid"  .= uuid
-    , "method"        .= ts (requestMethod req)
-    , "url"           .= ts (rawPathInfo req)
-    , "response_body" .= responseBody
-    , "status" .= object [ "code"    .= statusCode (responseStatus resp)
-                         , "message" .= ts (statusMessage (responseStatus resp))
-                         ]
-    , "time"   .= object [ "full"    .= full time
-                         , "process" .= processing time
-                         ]
+defaultLogResponse uuid req resp responseInfo time =
+    [ "request_uuid"     .= uuid
+    , "method"           .= ts (requestMethod req)
+    , "url"              .= ts (rawPathInfo req)
+    , "response_info"    .= responseInfo
+    , "response_code"    .= statusCode (responseStatus resp)
+    , "response_message" .= ts (statusMessage (responseStatus resp))
+    , "full_time"        .= full time
+    , "processing_time"  .= processing time
```

Comment:
_Nitpick:_ `time_full` and `time_processing` instead?
Not sure if that makes it easier to search for the field in ES/Kibana.